trusted-firmware-m/platform/ext/target/nordic_nrf/common/nrf5340/target_cfg.c

```diff
@@ -196,6 +196,10 @@ enum tfm_plat_err_t spu_init_cfg(void)
         memory_regions.non_secure_partition_limit);
     spu_regions_sram_config_non_secure(NS_DATA_START, NS_DATA_LIMIT);
 
+#if defined(NS_STORAGE_OFFSET) && defined(NS_STORAGE_LIMIT)
```

Comment:
Check whether the size is non-zero as well.

Reply:
Done